### ISSUE TITLE
Monitoring Site

### ISSUE THREAD

**Issue by Yamanipanuchi** [RainbowMiner/RainbowMiner#189]
None of my miners show up on the monitoring site. Should I be seeing information there by now?

**Comment by RainbowMiner** [comment]
have you enabled the monitoring?

Check the Config\config.txt. These three values need to be set:
"EnableMinerStatus": "1",
"MinerStatusUrl": "https:\\rbminer.net",
"MinerStarusKey": "youruniquekey",

Just to be sure: would you please send your config.txt and the Logs\Rainbowminer_2018-10-01.txt? Then I will have a look.

**Comment by RainbowMiner** [comment]
Yes, the information should be seen at once.

**Comment by RainbowMiner** [comment]
Do you get any warnings?

**Comment by Yamanipanuchi** [comment]
[config.txt](https://github.com/RainbowMiner/RainbowMiner/files/2436101/config.txt)
[RainbowMiner_2018-10-01.txt](https://github.com/RainbowMiner/RainbowMiner/files/2436102/RainbowMiner_2018-10-01.txt)

This miner is the only one have easy access to right now... It does not mine all the time.. But has the same config and I have not seen it.

**Comment by Yamanipanuchi** [comment]
Made a new key and it still did work... 

https://rbminer.net/monitoring/?user=5e376814-abdf-4fa8-b111-480b3f528956

**Comment by RainbowMiner** [comment]
Ah! Ok, I found it. Since no key and url is setup in your config.txt, I got curious: I forgot to set those fields in the build in configuration. It will be fixed in the next release v3.8.8.6
You can go ahead and put the values directly into the config.txt, if you want:

Open config.txt with notepad and find those two lines:
```
  "MinerStatusUrl": "$MinerStatusUrl",
  "MinerStatusKey": "$MinerStatusKey",
```

then replace the $.. variables like that:
```
  "MinerStatusUrl": "https://rbminer.net",
  "MinerStatusKey": "5e376814-abdf-4fa8-b111-480b3f528956",
```

.. then, you will be fine :)
